frontend/src/modules/Organizations/components/OrganizationTeamInviteEditForm.js

```diff
@@ -47,35 +144,31 @@ export const OrganizationTeamInviteEditForm = (props) => {
           of. All members of that group will be able to access your environment.
         </Typography>
         <Box sx={{ p: 3 }}>
-          <CardContent>
-            <TextField
-              disabled
-              fullWidth
-              label="Team"
-              name="team"
-              value={team.groupUri}
-              variant="outlined"
-            />
-          </CardContent>
           <CardContent>
             <Paper>
-              <CardHeader title="Organization Permissions" />
+              <CardHeader title="Additional Permissions" />
               <Divider />
               <CardContent sx={{ ml: 2 }}>
-                {permissions.length > 0 ? (
-                  permissions.map((perm) => (
+                {selected_permissions.length > 0 ? (
+                  selected_permissions.map((perm) => (
                     <Box>
                       <FormGroup>
                         <FormControlLabel
                           color="primary"
                           control={
                             <Switch
-                              disabled
-                              defaultChecked
+                              checked={switchState[perm.name]}
                               color="primary"
                               edge="start"
                               name={perm.name}
                               value={perm.name}
+                              onChange={(evt) => {
+                                perm.selected = evt.target.checked;
+                                const switchState_tmp = { ...switchState };
+                                switchState_tmp[perm.name] = perm.selected;
+                                setSwitchState(switchState_tmp);
```

Comment:
I'm curious on how this is implemented so differently from `EnvironmentTeamInviteEditForm.js`, how come?

Reply:
I built into on already existing front. without any massive changes. 
If you want, we can unify it, but, i'm afraid it's a story for a big separate PR